### PR TITLE
Changelog methode update

### PR DESCRIPTION
**Pull request by Furtif**

### CODE REVIEW

File: PoGo.NecroBot.Logic/State/VersionCheckState.cs [hunk 1]
```diff
@@ -29,6 +29,9 @@ public class VersionCheckState : IState
         public const string RemoteReleaseUrl =
             "https://github.com/Necrobot-Private/NecroBot/releases/download/v";
 
+        public const string ChangelogUri =
+             "https://cdn.rawgit.com/Necrobot-Private/NecroBot/master/CHANGELOG.md";
```
Comment (CDAGaming) [inline PoGo.NecroBot.Logic/State/VersionCheckState.cs]: If it's erroring on yours, I'd recommend changing this to raw.githubusercontent.com as although it doesn't error for most, it'll ensure max compatibility

Comment (Furtif) [inline PoGo.NecroBot.Logic/State/VersionCheckState.cs]: ok ok i change it

**Comment by CDAGaming** [comment]
Approved for Merge!

**Comment by CDAGaming** [comment]
Changelog Commits will be coming today to improve upon it's format

**Pull request merged**